BR-Restore2EC2/DR2EC2.ps1

```diff
@@ -0,0 +1,48 @@
+﻿##
+## Automated restore to EC2
+## By Dustin Albertson @ Veeam
+##
+
+#Please Update the variables below 
+$vbrserver = “localhost”
+# VBR ServerName
+$vbruser = “USERNAME”
+#VBR Username
+$vbrpwd = “PASSWORD”
+#VBR Password
+$accesskey = "ACCESSKEY"
+#Access Key from Cloud Credential in VBR (IE the cloud account you wish to use)
+$cust_csv = "C:/example/dr2ec2.csv"
+#Location of the csv file with the AWS/vm details
+
+## Script Below Edit at your own risk 
+
+$csv = import-csv $cust_csv
+asnp “VeeamPSSnapIn” -ErrorAction SilentlyContinue
+Connect-VBRServer #-Server $vbrserver -User $vbruser -Password $vbrpwd
+
+
+    $csv | ForEach-Object {
+    $vm_name = $_.VM_NAME
+    $vm_instance = $_.instance_size
+    $vm_region = $_.Region
+    $vm_az = $_.AZ
+    $vm_vpc = $_.VPC_ID 
+    $sgname = $_.SecurityGroupName
+    $sn_cidr = $_.SubnetCIDR
+        
+    $restorepoint = Get-VBRRestorePoint -Name $vm_name | Sort-Object –Property CreationTime –Descending | Select -First 1
+    $account = Get-VBRAmazonAccount -accesskey $accesskey
+    $region = Get-VBRAmazonEC2Region -Account $account -RegionType Global -Name $vm_region
+    $vm_disk = Get-VBRFilesInRestorePoint -RestorePoint $restorepoint | Where FileName -Like ‘*flat.vmdk*'
```

Comment:
I believe there's a typo here. Looks like one of the single quotes isn't a single quote.

Reply:
No typo,  this functions correctly as is.